common/common-client/src/main/java/org/ow2/proactive/authentication/iam/IAMRestClient.java

```diff
@@ -67,17 +68,17 @@ public String getSSOTicket(String url, String username, String password, String
                 throw new IAMException("Failed to acquire SSO ticket for the user" + username);
             }
 
-            Header[] headers = httpResponse.getHeaders(ticketHeader);
+            String htmlResponse = EntityUtils.toString(httpResponse.getEntity());
+            Document doc = Jsoup.parse(htmlResponse);
+            String tgtAttribute = doc.getElementsByAttribute(ticketMarker).first().attributes().get(ticketMarker);
```

Comment:
tgtAttribute ? It is not clear for me maybe you could add some comment.

Reply:
OK, I'll change it to 'htmlSSOTicket'. When we send a REST request to IAM to get a SSO ticket, it responds in a html format. So we parse the reponse to get a html SSO ticket, then we extract the final ticket from that 'htmlSSOTicket'.